crdsonnet/render.libsonnet

```diff
@@ -1,9 +1,11 @@
+local jutils = import 'github.com/Duologic/jsonnet-libsonnet/utils.libsonnet';
 local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
 
 {
-  local engineTypes = {
-    static: import 'static.libsonnet',
-    dynamic: import 'dynamic.libsonnet',
+  local engines = {
+    static: import 'renderEngines/static.libsonnet',
+    dynamic: import 'renderEngines/dynamic.libsonnet',
+    ast: import 'renderEngines/ast.libsonnet',
```

Comment:
You could make the names constants (locals if they are only used in this file, or in their own file) and refer to them by variable instead of referring to strings in many locations

Reply:
Fair

Reply:
done